app/models/base.rb

```diff
@@ -1,5 +1,22 @@
 class Base < JsonApiClient::Resource
+  # This connection class grabs the bearer token from the thread key-value
+  # store and adds it to our connection. This token was saved there for us when
+  # the user was first authenticated.
+  class MCBConnection < JsonApiClient::Connection
+    def run(request_method, path, params: nil, headers: {}, body: nil)
+      authorization = "Bearer #{Thread.current[:manage_courses_backend_token]}"
```

Comment:
Should we throw something up to Sentry if this lookup finds nil?

Reply:
Ya, good call! We should be using `.fetch`.